Tests/SwiftSyntaxBuilderTest/ClassDeclSyntaxTests.swift

```diff
@@ -35,7 +35,7 @@ final class ClassDeclSyntaxTests: XCTestCase {
   }
 
   func testThrowableClass() throws {
-    let buildable = try ClassDeclSyntax(name: .identifier("Foo")) {
+    let buildable = try ClassDeclSyntax(classHeader: .init(name: .identifier("Foo"))) {
```

Comment:
In all of these test cases, I would prefer to not write `.init`. It’s easier to write but makes you wonder which type the header node has when reading and that’s essential information for tests that test the syntax tree structure.